docs/accessibility.md

```diff
@@ -1,65 +1,5 @@
 ---
 permalink: /accessibility/
 title: Accessibility
-sidenav:
-  - text: Our commitment
-    href: "#our-commitment"
-  - text: Standards
-    href: "#standards"
-  - text: Evaluation
-    href: "#evaluation"
-  - text: Known limitations
-    href: "#known-limitations"
+redirect_to: https://login.gov/accessibility/
 ---
-<div class="usa-alert usa-alert--warning">
-  <div class="usa-alert__body">
-    <p class="usa-alert__text">This page contains draft content and is under review.
-</p>
-  </div>
-</div>
-
-## Our commitment
-Login.gov is committed to be inclusive and accessible for all of our users. Our focus is to continuously improve how people of all abilities are able to use Login.gov to easily access government applications and services.
-
-## Standards
-Login.gov aims to comply with the [W3C Web Content Accessibility Guidelines (WCAG) version 2.1 level AA](https://www.w3.org/TR/WCAG21/).
-
-You should be able to do the following with Login.gov:
-- Use a screen reader and read the content of the page by using the down arrow key.
--  Be able to see and hear content including separating foreground from background
-- You can change font size easily using your browser. (Please refer to your browser’s Help menu to learn how to re- size text.)
-
-## Evaluation
-Login.gov is regularly evaluated through a series of semi-automated evaluation tools and manual evaluation by expert reviewers. After each evaluation, we develop a plan to address any areas of improvement.
-
-Our completed accessibility assessment reports can be found below:
-
-[IAL 1 Account creation report](https://docs.google.com/document/d/1wqbVOvNg-7tFW1l4ZxMJrA_VR2gQqNpimlbzxPtvpXg/export?format=pdf)
-
-### Accessibility policies
-
-In addition to completed accessibility reports, Login.gov has compiled a page regarding accessibility policies.
-
-[Login.gov accessibility policies]({% link accessibility/policies.md %})
```

Comment:
Do we want to remove this page as well?

https://design.login.gov/accessibility/policies/

Since we don't link to it anywhere now.